nixos/modules/services/backup/bacula.nix

```diff
@@ -6,15 +6,14 @@
 with lib;
 
 let
-  libDir = "/var/lib/bacula";
```

Comment:
This was the original directory that systemd insists on taking ownership of and setting as root.  This isn't going to work because the applications works as the user "bacula" so can't read and write to it.